Makefile

```diff
@@ -0,0 +1,64 @@
+DOCKER_REPO ?= quay.io/observatorium/observatorium
+VERSION := $(strip $(shell [ -d .git ] && git describe --always --tags --dirty))
+BUILD_DATE := $(shell date -u +"%Y-%m-%dT%H:%M:%S%Z")
+VCS_REF := $(strip $(shell [ -d .git ] && git rev-parse --short HEAD))
+
+CONTAINER_CMD:=docker run --rm \
+		-u="$(shell id -u):$(shell id -g)" \
+		-v "$(shell go env GOCACHE):/.cache/go-build" \
+		-v "$(PWD):/go/src/github.com/observatorium/observatorium:Z" \
+		-w "/go/src/github.com/observatorium/observatorium" \
+		-e USER=deadbeef \
+		-e GO111MODULE=on \
+		quay.io/coreos/jsonnet-ci
+
+default: observatorium
+all: clean lint test observatorium
+
+tmp/help.txt: clean build
+	mkdir -p tmp
+	-./observatorium --help &> tmp/help.txt
+
+README.md: tmp/help.txt
+	embedmd -w README.md
+
+observatorium: vendor main.go $(wildcard *.go) $(wildcard */*.go)
```

Comment:
I guess `build` might be more straightforward?

Reply:
We also have `build`. The distinction is, one of them specifies os and arch, and builds a binary to be used in the container. I can remove `observatorium` if it's creating confusion, in upcoming PRs.

Reply:
It's fine